rest-api/dao/participant_summary_dao.py

```diff
@@ -322,6 +375,11 @@ def update_enrollment_status(self, summary):
                                                          summary.numCompletedBaselinePPIModules,
                                                          summary.physicalMeasurementsStatus,
                                                          summary.samplesToIsolateDNA)
+    summary.enrollmentStatusMemberTime = self.calculate_member_time(consent, summary)
+    summary.enrollmentStatusCoreOrderedSampleTime = self.calculate_core_ordered_sample_time(
```

Comment:
Update member time, ordered time and stored time in update_enrollment_status, so that when questionnaire response/PM/BO comes, it can be called and set those timestamps.